src/app/scripts/cac/cac-isochrone.js

```diff
@@ -0,0 +1,58 @@
+CAC.Pages.Home = (function ($) {
+    'use strict';
+
+    var payload = {};
+    var isochroneUrl = '/reachable';
+    var module = {
+        payload: payload,
+        fetchReachable: fetchReachable,
+        testFetch: testFetch
+    };
+
+    return module;
+
+    /**
+     * Use test values to attempt a call to the isochrone endpoint
+     *
+     * @return undefined Use side effects to print data out
+     */
+    function testFetch() {
+        var testParams = {
+            coords: {
+                lat: 39.954688,
+                lng: -75.204677
+            },
+            mode: ['WALK', 'TRANSIT'],
+            date: '01-21-2015',
+            time: '7:30am',
+            maxTravelTime: 5000,
+            maxWalkDistance: 5000
+        };
+        fetchReachable(testParams).then(function(data) {
+            console.log(data);
+        });
+    }
+
+    /**
+     * Fetch all the reachable destinations within our destination database
+     *
+     * @return {object} (promise) which will ultimately be a list of reachable destinations
+     */
+    function fetchReachable() {
+        if (payload.coords && payload.mode && payload.date &&
+            payload.time && payload.maxTravelTime && payload.maxWalkDistance) {
+            var urlParams = $.param(payload);
+            var requestUrl = isochroneUrl + urlParams;
+            var promise = $.ajax({
+                type: 'GET',
+                url: requestUrl,
+                contentType: 'application/json'
+            });
+            return promise;
+        } else {
+            console.log("INCOMPLETE PAYLOAD: UNABLE TO CALCULATE ISOCHRONE")
+            return false;
```

Comment:
For consistency and usability, you should create a promise here and reject it with an error message or object.

See jQuery deferred object: http://api.jquery.com/category/deferred-object/